lib/katello/engine.rb

```diff
@@ -155,6 +155,7 @@ class Engine < ::Rails::Engine
       # Lib Extensions
       ::Foreman::Renderer::Scope::Variables::Base.include Katello::Concerns::RendererExtensions
       ::Foreman::Renderer::Scope::Base.include Katello::Concerns::BaseTemplateScopeExtensions
+      ::Foreman::Renderer::Configuration.include Katello::Concerns::RendererConfigurationExtension
```

Comment:
Perhaps i did not explain myself well, what I suggested was changing this to 
```suggestion
      ::Foreman::Renderer::Configuration.prepend Katello::Concerns::RendererConfigurationExtension
```

And changing the new module to:
```ruby
module Katello
  module Concerns
    module RendererConfigurationExtension
      def allowed_helpers
        super + [:errata, :host_subscriptions, :host_applicable_errata_ids, :host_applicable_errata_filtered,
                 :host_latest_applicable_rpm_version, :load_pools, :load_errata_applications, :host_content_facet,
                 :host_sla, :host_products, :sub_name, :sub_sku, :registered_through, :last_checkin, :host_collections,
                 :host_subscriptions_names, :host_subscriptions, :host_products_names, :host_collections_names,
                 :host_redhat_subscription_names, :registered_at]
      end
    end
  end
end
```

However, looking at the Foreman plugin registry, I think this monkey patching is not actually needed at all - we can use the `allowed_template_helpers` method to register these helpers:

https://github.com/theforeman/foreman/blob/develop/app/registries/foreman/plugin.rb#L367-L374